jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_ECEN

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+source "${HOMEgfs}/ush/preamble.sh"
+export WIPE_DATA="NO"
+export DATA="${DATAROOT}/${RUN}ocnanal_${cyc}"
+source "${HOMEgfs}/ush/jjob_header.sh" -e "ocnanalecen" -c "base ocnanalecen"
+
+
+##############################################
+# Set variables used in the script
+##############################################
+
+##############################################
+# Begin JOB SPECIFIC work
+##############################################
+
+export PYTHONPATH=${HOMEgfs}/sorc/gdas.cd/ush:${PYTHONPATH}
+
+###############################################################
+# Run relevant script
+
+EXSCRIPT=${GDASOCNRUNSH:-${HOMEgfs}/sorc/gdas.cd/scripts/exgdas_global_marine_analysis_ecen.py}
```

Comment:
This is not a shell script.
Also, `GDASOCNRUNSH` is **not** the variable for this script.  It is used in the `ocnanalrun`

Reply:
oof, changed.